spec/src/lib.rs

```diff
@@ -99,7 +99,7 @@ pub struct Genesis {
     pub compact_target: u32,
     pub uncles_hash: H256,
     pub hash: Option<H256>,
-    pub nonce: U128,
+    pub nonce: u128,
```

Comment:
https://github.com/nervosnetwork/ckb/blob/develop/resource/specs/dev.toml#L9

Should `"0x0"` -> `0` ? And integration.toml as well.

Reply:
I didn't notice that. now fixed, thanks.

Reply:
`toml-rs` is not supported parse u128 now

Reply:
WTH...